recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -14,11 +14,11 @@ class PahoMqttcConan(ConanFile):
     options = {"shared": [True, False],
                "fPIC": [True, False],
                "ssl": [True, False],
-               "asynchronous": [True, False]}
+               "samples": [True, False]}
```

Comment:
why do you need sample, if the idea is only package the library?

Reply:
The samples included the command line utilities that use the library.  Would those not be useful?

Reply:
Yes, they are useful, I remember that I used it. Keep it as optional, but disabled.

Reply:
Samples option included, default False.

Reply:
Samples option included.  Default False.